src/main/java/uk/gov/companieshouse/confirmationstatementapi/interceptor/FilingInterceptor.java

```diff
@@ -26,11 +24,11 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         }
 
         if (TransactionStatus.CLOSED.equals(transaction.getStatus())) {
-            LOGGER.debug("{} closed proceeding to generate filing", transaction.getId());
+            LOGGER.debug(String.format("%s closed proceeding to generate filing", transaction.getId()));
             return true;
         }
 
-        LOGGER.info("{} not closed rejecting filing request", transaction.getId());
+        LOGGER.debug(String.format("%s not closed rejecting filing request", transaction.getId()));
```

Comment:
we can use debugContext here similar to how we did the TransactionInterceptor, we have the request passed in as a method param.